Wrong instructions in readme for mvn exec:java

I think we have switched to `exec:exec`
```
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:java (default-cli) on project eventsourced-customer-registry: Unable to parse configuration of mojo org.codehaus.mojo:exec-maven-plugin:3.0.0:java for parameter arguments: Cannot store value into array: arraycopy: element type mismatch: can not cast one of the elements of java.lang.Object[] to the type of the destination array, java.lang.String
```

--- a/samples/java-eventsourced-customer-registry/README.md
+++ b/samples/java-eventsourced-customer-registry/README.md
@@ -7,18 +7,12 @@ To run the example locally:
 * Start the example:
   * publish relevant projects for use from Maven
     ```
-    sbt publishM2
-    ```
-  * build the latest Maven plugin and archetypes
-    ```
-    cd maven-java
-    mvn versions:set -DnewVersion="0.7.0-beta....-SNAPSHOT"
-    mvn install
+    ./publishLocalM2.sh
     ```
   * trigger codegen, compile and run from Maven
     ```
     cd samples/java-eventsourced-customer-registry
-    mvn -Dakkaserverless-sdk.version="0.7.0-beta....-SNAPSHOT" compile exec:java
+    mvn -Dakkaserverless-sdk.version=0.7.0-beta....-dev-SNAPSHOT compile exec:exec
     ```
 
 * Start the proxy

--- a/samples/java-first-service/README.md
+++ b/samples/java-first-service/README.md
@@ -37,7 +37,6 @@ To start the application locally, the `exec-maven-plugin` is used. Use the follo
 
 ```
 mvn compile exec:exec
-```ile exec:java
 ```
 
 With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://developer.lightbend.com/docs/akka-serverless/java/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:

--- a/samples/java-valueentity-customer-registry/README.md
+++ b/samples/java-valueentity-customer-registry/README.md
@@ -7,18 +7,12 @@ To run the example locally:
 * Start the example:
   * publish relevant projects for use from Maven
     ```
-    sbt publishM2
-    ```
-  * build the latest Maven plugin and archetypes
-    ```
-    cd maven-java
-    mvn versions:set -DnewVersion="0.7.0-beta....-SNAPSHOT"
-    mvn install
+    ./publishLocalM2.sh
     ```
   * trigger codegen, compile and run form Maven
     ```
     cd samples/java-valueentity-customer-registry
-    mvn -Dakkaserverless-sdk.version="0.7.0-beta....-SNAPSHOT" compile exec:java
+    mvn -Dakkaserverless-sdk.version=0.7.0-beta....-dev-SNAPSHOT compile exec:exec
     ```
 
 * Start the proxy

